pages/landing/hipaa/template/pricing/index.html

```diff
@@ -18,31 +18,31 @@
     description:
       Early access subscribers will receive a CIS staging environment when invited<br>
       and a fully HIPAA-compliant production infrastructure when it’s ready.
-    price: 3060
-    strike_price: 10000
-    flat_fee: 7425
-    strike_flat_fee: 9900
+    price: "3,060"
+    strike_price: "10,000"
```

Comment:
It looks like a key change is we no longer show the original price... So the customer has no way of knowing how much of a discount they are getting... That seems to weaken the "discount" proposition, doesn't it?

Reply:
I do not have a strong view on this. This change is in response to [Josh's comments on Slack](https://gruntwork-io.slack.com/archives/C0275GS2FKM/p1626909175224300?thread_ts=1626879024.181400&cid=C0275GS2FKM). If we want to keep the crossed out pricing that's fine.

Reply:
Updated in 83985751.